lua/gitlab/reviewer/diffview.lua

```diff
@@ -100,8 +105,11 @@ M.get_location = function(range)
     u.notify("Diffview reviewer must be initialized first")
     return
   end
+
   local bufnr = vim.api.nvim_get_current_buf()
-  local current_line = vim.api.nvim_win_get_cursor(0)[1]
+
+  -- If there's a range, use the start of the visual selection, not the current line
+  local current_line = range and range.start_line or vim.api.nvim_win_get_cursor(0)[1]
```

Comment:
Use the top of the visual selection as the current line, not the bottom, when there is a range